webpack.config.js

```diff
@@ -10,6 +10,14 @@ module.exports = {
 	entry: ['./demo/tc-ui/browser/main.js'],
 	resolve: {
 		extensions: ['.js', '.jsx', '.css'],
+		alias: {
+			react: 'preact/compat',
+			'react-dom': 'preact/compat',
+			// Not necessary unless you consume a module using `createClass`
+			'create-react-class': 'preact/compat/lib/create-react-class',
+			// Not necessary unless you consume a module requiring `react-dom-factories`
+			'react-dom-factories': 'preact/compat/lib/react-dom-factories',
+		},
```

Comment:
👍 

I remember once biz-ops-admin UI had a bundling JS file issue that transpile via babel won't raise error although missing component and some cases, so installing webpack will solve that concern.